protected/humhub/libs/BaseSettingsManager.php

```diff
@@ -153,6 +154,12 @@ public function getSerialized(string $name, $default = null, bool $asArray = tru
      */
     public function get(string $name, $default = null)
     {
+        [$name, $moduleId] = Setting::fixModuleIdAndName($name, $this->moduleId);
+
+        if ($moduleId != $this->moduleId) {
```

Comment:
Can you please explain these line?

It's for the case, when the if the Module Id was renamed?
But how would reload Values solves this problem?